OneCommunityGlobal/HGNRest
# yihan fix weekly summary submission date

# Description
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/94303659/0de0c3df-94a0-494b-af2f-0b3d8f098086)

## Related PRS (if any):
This backend PR is related to the[ #840](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/840) frontend PR.

## Main changes explained:
Added uploadDate in weeklySummaries object. 

## How to test:
check into current branch on both frontend and backend
do `npm install` and  `npm run build`  to run this PR locally
then follow the instruction of PR of frontend to test
